openmetadata-ui/src/main/resources/ui/src/pages/GlossaryPage/GlossaryPageV1.component.tsx

```diff
@@ -482,6 +482,22 @@ const GlossaryPageV1 = () => {
     return patchGlossaries(selectedData?.id as string, jsonPatch);
   };
 
+  const handleRedirection = (fqn?: string) => {
+    const redirectFqn = fqn
+      ? fqn
+      : selectedKey.split('.').slice(0, -1).join('.');
+
+    if (isEmpty(redirectFqn)) {
+      setGlossariesList([]);
+      setIsLoading(true);
+      history.push(getGlossaryPath());
+      fetchGlossaryList();
+    } else {
+      history.push(getGlossaryPath(redirectFqn));
+      fetchGlossaryList(redirectFqn);
+    }
+  };
+
```

Comment:
Can we add some comments for this method?

Reply:
Hi @Sachin-chaurasiya method name is self-explanatory, its just redirects to glossary

Reply:
@ShaileshParmar11 , Can we add comments for this logic?
```
const redirectFqn = fqn
      ? fqn
      : selectedKey.split('.').slice(0, -1).join('.');
```